variables.tf

```diff
@@ -25,7 +25,13 @@ variable "enable_metrics" {
 
 variable "key_protect_instance_name" {
   type        = string
-  description = "The name of the existing Key Protect instance, if 'create_key_protect_instance' is true a new instance will be created with this name."
+  description = "The name of the Key Protect instance to provision."
```

Comment:
The name to give the Key Protect instance that will be provisioned by the module. Not used if value is passed for var.existing_key_protect_instance_guid